### PR TITLE
Add more debug logging to try and find out why ring and ringback sounds aren't playing

### PR DESCRIPTION
**Pull request by MadLittleMods**
Add more debug logging to try and find out why ring and ringback sounds aren't playing

To better investigate https://github.com/vector-im/element-web/issues/20832

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

File: src/CallHandler.tsx [hunk 1]
```diff
@@ -400,20 +404,30 @@ export default class CallHandler extends EventEmitter {
             } else {
                 this.audioPromises.set(audioId, playAudio());
             }
+        } else {
+            logger.warn(`${logPrefix} Unable to find <audio> element for ${audioId}`);
```
Comment (SimonBrandner) [inline src/CallHandler.tsx:408]:
```suggestion
            logger.warn(`${logPrefix} unable to find <audio> element for ${audioId}`);
```

File: src/CallHandler.tsx [hunk 2]
```diff
@@ -400,20 +404,30 @@ export default class CallHandler extends EventEmitter {
...
             if (this.audioPromises.has(audioId)) {
-                this.audioPromises.set(audioId, this.audioPromises.get(audioId).then(() => audio.pause()));
+                this.audioPromises.set(audioId, this.audioPromises.get(audioId).then(pauseAudio));
             } else {
-                // pause doesn't return a promise, so just do it
-                audio.pause();
+                pauseAudio();
             }
+        } else {
+            logger.warn(`${logPrefix} Unable to find <audio> element for ${audioId}`);
```
Comment (SimonBrandner) [inline src/CallHandler.tsx:430]:
```suggestion
            logger.warn(`${logPrefix} unable to find <audio> element for ${audioId}`);
```

**Review by SimonBrandner — APPROVED** [review]
Looks sane! (just a few consistency nitpicks)

**Comment by MadLittleMods** [comment]
Thanks for the review @SimonBrandner 🐆

**Comment by SimonBrandner** [comment]
> Thanks for the review @SimonBrandner 🐆

No problem at all!

**Pull request merged**